e2e-test/e2e/chaos/stresschaos/memory.go

```diff
@@ -45,7 +50,7 @@ func TestcaseMemoryStressInjectionOnceThenRecover(
 		if err != nil {
 			return false, err
 		}
-		if conditions[0].MemoryUsage-conditions[1].MemoryUsage > 50*1024*1024*0.9 {
+		if conditions[0].MemoryUsage-conditions[1].MemoryUsage > allowedJitter {
```

Comment:
Should that be `conditions[0].MemoryUsage-conditions[1].MemoryUsage > 50*1024*1024-allowedJitter` ?